Code/Framework/AzQtComponents/AzQtComponents/Components/Widgets/BreadCrumbs.h

```diff
@@ -155,6 +163,14 @@ namespace AzQtComponents
         QStringList m_truncatedPaths;
         AZ_POP_DISABLE_WARNING
         bool m_pushPathOnLinkActivation = true;
+        int m_currentPathSize = 0;
+
+        QString m_defaultIcon;
+        AZ_PUSH_DISABLE_WARNING(
+            4251, "-Wunknown-warning-option") // 4251: 'AzQtComponents::BreadCrumbs::m_currentPathIcons': class 'QVector<QIcon>' needs to have
+                                              // dll-interface to be used by clients of class 'AzQtComponents::BreadCrumbs'
+        QVector<QString> m_currentPathIcons;
```

Comment:
Nitpick: You could use `QStringList` instead to avoid the `QVector` warning disable

Reply:
I tested it out, and I get the same warning with QStringList. Also, using a list over a vector would have required some changes to the logic. So I left it as-is in the end.